packages/suite/src/support/messages.ts

```diff
@@ -4409,6 +4442,10 @@ export default defineMessages({
         defaultMessage:
             'Your device is already updated to latest firmware. You may reinstall the firmware if needed.',
     },
+    TR_SWITCH_FIRMWARE_TYPE_DESCRIPTION: {
+        id: 'TR_SWITCH_FIRMWARE_TYPE_DESCRIPTION',
+        defaultMessage: 'You are going to install a different type of firmware.',
```

Comment:
better copy

Reply:
Improved description in de22367abb28e3d83e315041a0da6957ef04fdac.
Not sure if we should talk about U2F and password manager here or provide links to wiki @hynek-jina 
![Screenshot 2022-08-01 at 16 15 45](https://user-images.githubusercontent.com/42465546/182167896-e59c64d8-0489-46a7-847e-880a31887cc2.png)